### PR TITLE
Added SDG infographics, renamed layout folder to shared, updated World Goals page to show SDG overview information and infographics

### PR DESCRIPTION
**Pull request by dwnoble**
Desktop, top of the page:
Screenshot 2023-09-06 at 3 01 17 PM

Goal with an inforgraphic:
Screenshot 2023-09-06 at 3 01 24 PM

Goal with an infographic on a smaller screen size:
Screenshot 2023-09-06 at 3 01 33 PM

### CODE REVIEW

**Review by juliawu — APPROVED** [review]
Looks very nice!

(1) I think the spacing between cards is a little larger than what's on Figma. You might consider tweaking it a bit?
(2) We should standardize to a linter, looks like a lot of these changes are linting my changes -- sorry!

File: experimental/sdg-tracker/src/index.css [hunk 1]
```diff
@@ -35,6 +36,17 @@ body {
...
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif,
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
```
Comment (beets) [inline experimental/sdg-tracker/src/index.css:49]: whoops - sorry you'll need to overwrite the change i just submitted to this file (i moved it all including the @import statement)

**Review by beets — APPROVED** [review]
nice!

File: experimental/sdg-tracker/src/index.css (hunk 1 above)
Comment (dwnoble) [reply experimental/sdg-tracker/src/index.css:51]: no sweat! i merged and overwrote this w/your changes

**Comment by dwnoble** [comment]
> Looks very nice!
> 
> (1) I think the spacing between cards is a little larger than what's on Figma. You might consider tweaking it a bit? (2) We should standardize to a linter, looks like a lot of these changes are linting my changes -- sorry!

(1) fixed!
(2) agreed - not sure why that's happening. let's reconvene on this tomorrow

**Pull request merged**